index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<header id= "header">
+    <img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.collinsdictionary.com%2Fdictionary%2Fenglish%2Flake&psig=AOvVaw1e6qxrewvh4DFgrQ24h_Wi&ust=1639075797144000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCPCDqKfv1PQCFQAAAAAdAAAAABAD" alt="" id="header-img">
+    <nav id="nav-bar">
+
+        <button class="nav-link"> First button</button>
+        <button class="nav-link"> Second button</button>
+        <button class="nav-link"> Third button</button>
+        <button>Fourth button</button>
```

Comment:
Adding type attributes to html <button> tags would be more semantic. Kindly make the changes.